src/server/utils/validations.js

```diff
@@ -0,0 +1,68 @@
+'use strict'; // eslint-disable-line
+const moment = require('moment');
+const _ = require('lodash');
+
+const validations = {
+  /**
+   * Validate a string as and convert to a valid format.
+   * @param {string} inputDob The date of birth string to validate / convert.
+   * @returns {string} If input was valid a valid SSN if not valid an empty string.
+   */
+  dateOfBirth: (inputDob) => {
+    let convertedDob = '';
```

Comment:
I feel like you could elide this variable by just `return '';` in the error cases and returning the `moment()` call directly.